package.json

```diff
@@ -20,7 +20,7 @@
     "babel-core": "^6.24.1",
     "babel-loader": "^6.2.x",
     "babel-preset-es2015": "^6.24.1",
-    "backbone": "~1.1.2",
+    "backbone": "~1.3.3",
```

Comment:
Yeah, if you'd like,I'll move it to a separate PR.

However, I did review [the changelog](http://backbonejs.org/#changelog), and I don't think there will be any changes that we need to make to Girder. The separate PR would give this more visibility and scrutiny from other devs though.